### PR TITLE
make base-backgroud-compactions configurable

### PR DESCRIPTION
**Pull request by zhangjinpeng1987**
@lishihai9017 @siddontang PTAL

### CODE REVIEW

File: etc/config-template.toml [hunk 1]
```diff
@@ -113,6 +113,9 @@ max-background-compactions = 6
 # Maximum number of concurrent background flush jobs
 max-background-flushes = 2
 
+# Suggested number of concurrent background compaction jobs.
+# base-background-compactions = 1
```
Comment (siddontang) [inline etc/config-template.toml]: do we need to use a higher value by default?

Comment (zhangjinpeng1987) [inline etc/config-template.toml]: In normal cases 1 is enough, just in heavy write case we need increase it.

**Comment by zhangjinpeng1987** [comment]
@lishihai9017 PTAL

File: etc/config-template.toml (hunk 1 above)
Comment (siddontang) [inline etc/config-template.toml]: please add this to optimization doc later.

**Comment by hhkbp2** [comment]
LGTM

**Pull request merged**